aikau/src/main/resources/alfresco/search/AlfSearchList.js

```diff
@@ -468,7 +468,8 @@ define(["dojo/_base/declare",
        */
       updateFilterHash: function alfresco_search_AlfSearchList__updateFilterHash(fullFilter, mode) {
          // Get the existing hash and extract the individual facetFilters into an array
-         var aHash = hashUtils.getHash(),
+         var aHashString = hashUtils.getHashString(),
+             aHash = ioQuery.queryToObject(aHashString),
```

Comment:
No... we can't do this... read through the associated MNT issue... we need to support two versions of Share